manifests/repo.pp

```diff
@@ -17,17 +17,22 @@
         }
       }
       'Debian': {
-        $release_url = $facts['os']['name'] ? {
-          'Debian' => "http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_${major_release}.0",
-          'Ubuntu' => "http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_${major_release}",
+        $release_url_suffix = fact('os.name') ? {
```

Comment:
`fact()` a été ajouté dans la version 4.19.0 de la stdlib (https://github.com/puppetlabs/puppetlabs-stdlib/commit/740ca7dc8053be93e43392ca61b2f308c0596d19)

Le module utilise la version 4.6.0 avec la conf fixture-old.yml:
https://github.com/ccin2p3/puppet-syslog_ng/blob/master/.fixtures-old.yml#L5-L7

Reply:
Thanks, I was reading the wrong branch locally...